src/cljs/villagebookUI/fetchers.cljs

```diff
@@ -0,0 +1,24 @@
+(ns villagebookUI.fetchers
+  (:require [villagebookUI.store.user :as user-store]
+            [villagebookUI.api.user :as user-api]
+            [villagebookUI.store.organisations :as org-store]
+            [villagebookUI.api.organisation :as org-api]))
+
+
+(defn fetch-user! []
+  (user-api/get-data
+   (fn [res]
+     (user-store/add! res))
```

Comment:
`user-store/add!` only adds the user map to the state. `fetch-user!` makes a call to the user API and stores the result (a `user` map) in the state. Made this a separate function since this bit of code gets repeated at a number of places throughout the app.